rolling/distribution.yaml

```diff
@@ -1409,6 +1409,45 @@ repositories:
       url: https://github.com/ros2/mimick_vendor.git
       version: master
     status: maintained
+  moveit:
+    doc:
+      type: git
+      url: https://github.com/ros-planning/moveit2.git
+      version: main
+    release:
+      packages:
+      - moveit
+      - moveit_common
+      - moveit_core
+      - moveit_kinematics
+      - moveit_planners
+      - moveit_planners_ompl
+      - moveit_plugins
+      - moveit_ros
+      - moveit_ros_benchmarks
+      - moveit_ros_move_group
+      - moveit_ros_occupancy_map_monitor
+      - moveit_ros_perception
+      - moveit_ros_planning
+      - moveit_ros_planning_interface
+      - moveit_ros_robot_interaction
+      - moveit_ros_visualization
+      - moveit_ros_warehouse
+      - moveit_runtime
+      - moveit_servo
+      - moveit_simple_controller_manager
+      - run_move_group
+      - run_moveit_cpp
+      - run_ompl_constrained_planning
+      tags:
+        release: release/rolling/{package}/{version}
+      url: https://github.com/moveit/moveit2-release.git
```

Comment:
You should be aware that since the release repository is not in the `ros2-gbp` organization, this package will *not* be automatically forked off for the next stable distribution (Humble).  As long as you are OK with that state of affairs, I'm happy to merge it.

Reply:
With that you mean that there won't be a new track being generated for Humble? Or are there any other implications? If no, I'm ok with just merging this, or rather the [2.2.1 patch](https://github.com/ros-planning/moveit2/pull/552) that we're preparing right now.

Reply:
> With that you mean that there won't be a new track being generated for Humble?

That's correct, when we branch off for Humble, we won't automatically branch and create a new track in the release repository for this package.  There are no other implications, so if you are OK with that limitation, then I'll go ahead and merge this.

Reply:
Thanks!